app/celer-sim/simple-driver.py

```diff
@@ -42,6 +42,8 @@
     'optical': {
         'absorption': True,
         'rayleigh_scattering': True,
+        'wavelength_shifting': "exponential",
+        'wavelength_shifting2': "exponential"
```

Comment:
These parameter names are somewhat confusing, as they don’t clearly reflect the underlying time structure of remission. Names like wls_delay_time or wls_remission_time might sound better for their meaning (?).  

I assumed these parameters were intended for use in the WavelengthShiftGenerator when sampling time, which doesn't appear to be implemented yet - or am I missing something?

Reply:
No problem merging it as is and  extending the time sampling in a follow-up MR. Thanks!